builds/azure-pipelines/build-pr.yml

```diff
@@ -46,4 +46,4 @@ stages:
         configuration: '$(configuration)'
         nugetVersion: '99.99.99-test'
         binariesVersion: '99.99.99'
-        oopWorkerSupportedExtensionVersion: '99.99.99'
\ No newline at end of file
+        oopWorkerSupportedExtensionVersion: '99.99.99-test'
```

Comment:
The worker extension version is the nuget package version, not the binary version.